configure.ac

```diff
@@ -608,6 +621,7 @@ AC_MSG_RESULT([
   ${PACKAGE_NAME} ${VERSION}
 
   platform:                  $my_htop_platform
+  os-release file:           $with_os_release
   (Linux) proc directory:    $with_proc
```

Comment:
The label for proc is used mainly on Linux, but checked on all platforms (except WinNT). Maybe drop platform marker?